hack/acm-hub.sh

```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+set -x
+CLUSTER_NAME=${1:? "\$1 should be a cluster name in the standard FQDN format"}
+
+WORK_DIR=$HOME/.acm/"$CLUSTER_NAME"
+rm -rf "$WORK_DIR"
+mkdir -p "$WORK_DIR"
+
+cat <<EOF >"$WORK_DIR"/managed-cluster.yaml
+apiVersion: cluster.open-cluster-management.io/v1
+kind: ManagedCluster
+metadata:
+  name: "$CLUSTER_NAME"
+spec:
+  hubAcceptsClient: true
+EOF
+
+cat <<EOF >"$WORK_DIR"/klusterlet-addon-config.yaml
+apiVersion: agent.open-cluster-management.io/v1
+kind: KlusterletAddonConfig
+metadata:
+  name: "$CLUSTER_NAME"
+  namespace: "$CLUSTER_NAME"
+spec:
+  clusterName: "$CLUSTER_NAME"
+  clusterNamespace: "$CLUSTER_NAME"
+  applicationManager:
+    enabled: true
+  certPolicyController:
+    enabled: true
+  clusterLabels:
+    cloud: auto-detect
+    vendor: auto-detect
+  iamPolicyController:
+    enabled: true
+  policyController:
+    enabled: true
+  searchCollector:
+    enabled: true
+  version: 2.2.0
+EOF
+
+oc new-project "$CLUSTER_NAME" 1>/dev/null
+oc label namespace "$CLUSTER_NAME" cluster.open-cluster-management.io/managedCluster="$CLUSTER_NAME"
+oc apply -f "$WORK_DIR"/managed-cluster.yaml
+oc apply -f "$WORK_DIR"/klusterlet-addon-config.yaml
+sleep 3
```

Comment:
why sleep? can we do wait and watch?

Reply:
We definitely can enhance these scripts. Given the time crunch, I wanted to keep it simple.